library/build.gradle

```diff
@@ -1,15 +1,18 @@
 plugins {
     id 'com.android.library'
     id 'kotlin-android'
+    id 'maven-publish'
 }
 
+group = 'tachiyomiorg'
+
 android {
-    compileSdkVersion 30
-    buildToolsVersion "30.0.3"
+    compileSdkVersion 33
+    ndkVersion "25.1.8937393"
 
     defaultConfig {
-        minSdkVersion 21
-        targetSdkVersion 30
+        minSdkVersion 23
```

Comment:
Why the increase?

Reply:
matched with tachiyomi-app 's version.
no other reason

Reply:
fixed